mobile/app/(tabs)/index.tsx

```diff
@@ -1,8 +1,30 @@
+import React, { useContext, useState, useEffect } from 'react';
 import { StyleSheet, View, Text } from 'react-native';
+import UserSignUpForm from '../Components/UserSignUpForm';
 
 export default function HomeScreen() {
+  // const { user, onChangeUser } = useState('');
+  const [email, onChangeEmail] = React.useState('');
+  const [demographics, onChangeDemo] = React.useState('');
+  const [name, onChangeName] = React.useState('');
+  const [phoneNum, onChangePhoneNum] = React.useState('');
+  const [community, onChangeCommunity] = React.useState('');
+
+
+
+
+  function onChangeUser(email, demographics, name, phoneNum, community) {
+    onChangeEmail(email);
+    onChangeDemo(demographics);
+    onChangeName(name);
+    onChangePhoneNum(phoneNum);
+    onChangeCommunity(community);
+    console.log(email + " " + demographics + " " + name + " " + phoneNum + " " + community);
+  }
```

Comment:
Whenever you're creating a component, only add content that is relevant. This highlighted functionality can all reside in your created `UserSignUpForm.tsx` instead of being BOTH `HomeScreen` and `UserSignUpForm`.

It's redundant and unclear what the purpose is since the interaction isn't centralized to the component this is related to.

Reply:
resolved